src/index.js

```diff
@@ -0,0 +1,9 @@
+import './styles/reset.css';
+import './styles/common.css';
+
+import { Layout } from './components/Layout';
+import { MovieListView } from './view/movieListView';
+
+const app = document.getElementById('app');
+app.appendChild(Layout());
+app.addEventListener('DOMContentLoaded', MovieListView());
```

Comment:
```suggestion
app.appendChild(Layout());
MovieListView();
```

그런데 이렇게 작성하는거랑 어떤 차이가 있을까요?
두 번째로, app에 대해서 DOMContentLoaded 이벤트가 발생하긴 할까요?

발생하지 않는다면 그 이유는 무엇일까요?